File: ido_sdk_flutter/ido_sdk/protocol_sdk/protocol_channel/android/src/main/kotlin/com/idosmart/model/IDOBpMeasurementParamModel.kt
//
//  IDOBpMeasurementParamModel.kt
//  protocol_channel
//
//  Created by hedongyang on 2023/10/30.
//

package com.idosmart.model

import com.google.gson.GsonBuilder
import com.google.gson.annotations.SerializedName

///
/// Blood pressure measurement event number

open class IDOBpMeasurementParamModel(flag: Int) : IDOBaseModel {

    /// 1: Start measurement
    /// 2: End measurement
    /// 3: Get blood pressure data 
    @SerializedName("flag")
    var flag: Int = flag


    

    override fun toJsonString(): String {
        return GsonBuilder().create().toJson(this).toString()
    }
}
    

File: ido_sdk_flutter/ido_sdk/protocol_sdk/protocol_channel/ios/Classes/Model/IDOBpMeasurementParamModel.swift
//
//  IDOBpMeasurementParamModel.swift
//  protocol_channel
//
//  Created by hc on 2023/10/12.
//

import Foundation
/// Blood pressure measurement event number
@objcMembers
public class IDOBpMeasurementParamModel: NSObject, IDOBaseModel {
    /// 1: Start measurement
    /// 2: End measurement
    /// 3: Get blood pressure data 
    public var flag: Int
    
    enum CodingKeys: String, CodingKey {
        case flag
    }
    
    public init(flag: Int) {
        self.flag = flag
    }
    
    public func toJsonString() -> String? {
        if let jsonData = try? JSONEncoder().encode(self) {
            let jsonString = String(data: jsonData, encoding: .utf8)
            return jsonString
        }
        return nil
    }
}

